packages/f/fast_io/xmake.lua

```diff
@@ -0,0 +1,21 @@
+package("fast_io")
+    set_kind("library", {headeronly = true})
+    set_homepage("https://github.com/cppfastio/fast_io")
+    set_description("Significantly faster input/output for C++20")
+    set_license("MIT")
+
+    add_urls("https://github.com/cppfastio/fast_io.git")
+    add_urls("https://bitbucket.org/ejsvifq_mabmip/fast_io.git")
+    add_urls("https://gitee.com/qabeowjbtkwb/fast_io.git")
+
+    on_install(function (package)
+        os.cp("include/*", package:installdir("include"))
```

Comment:
```
        os.cp("include", package:installdir())
```

Reply:
> ```
>         os.cp("include", package:installdir())
> ```

It can't work. See https://github.com/cppfastio/fast_io/tree/master/include
```sh
"fast_io.h": No such file or directory
```

Reply:
They are equivalent. it works for me.

```
checking for xmake::fast_io ... fast_io 2023.1.28
{ 
  sysincludedirs = { 
    "/home/ruki/.xmake/packages/f/fast_io/2023.1.28/6d258046efe5456884cfda4f0652a8bb/include" 
  },
  version = "2023.1.28",
  license = "MIT" 
}

patching /home/ruki/.xmake/packages/f/fast_io/2023.1.28/6d258046efe5456884cfda4f0652a8bb/lib/pkgconfig/fast_io.pc ..
checking for gcc ... /usr/bin/gcc
checking for the c++ compiler (cxx) ... gcc
checking for /usr/bin/gcc ... ok
checking for flags (-fPIC) ... ok
> gcc "-fPIC"
checking for flags (-std=c++20) ... ok
> gcc "-std=c++20"
> /usr/bin/gcc -c -m64 -std=c++20 -isystem /home/ruki/.xmake/packages/f/fast_io/2023.1.28/6d258046efe5456884cfda4f0652a8bb/include -o /dev/shm/.xmake999/230128/_73AB94A5FB72433087EF80620F150860.o /dev/shm/.xmake999/230128/_4C4F997076DF4972BC6F0745DD98984A.cpp
checking for flags (-fdiagnostics-color=always) ... ok
> gcc "-fdiagnostics-color=always"
> checking for c++ includes(fast_io.h)
> checking for c++ snippet(test)
  => install fast_io 2023.1.28 .. ok
```

Reply:
oh, my bad. Fixed.